AGENTS.md

```
@@ -0,0 +1,705 @@
+              npm publish + GitHub Release 자동
+```
+
+**핵심 명령어:**
+```bash
+pnpm changeset        # 변경 내용 기록 (PR 낼 때)
+pnpm changeset version # 버전 범프 (CI 자동)
+pnpm changeset publish # npm 배포 (CI 자동)
+```
+
+**semver 규칙:**
+| 변경 종류 | 버전 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)